packages/core/src/types/schema.ts

```diff
@@ -34,13 +34,15 @@ export interface PartialDataTypeProps<
   required?: TRequired // @todo HATCH-417
   default?: PrimitiveType | (() => PrimitiveType) | null
   unique?: boolean
+  displayName?: string | undefined
```

Comment:
Actually was able to remove it since `PartialControlType` can use it. Thanks for catching this @roy-coder

Reply:
Think I spoke too soon. `displayName` needs to be a data type prop to meet AC. The displayName prop is expected to be used in the schema definition. It will overlap the same way default does. Unless it doesn't need to be in `PartialControlType`.